File: sdkwork-backend-sdk-kotlin/src/main/kotlin/com/sdkwork/backend/PlusOssBucketForm.kt
package com.sdkwork.backend

data class PlusOssBucketForm(
    val name: String? = null,
    val region: String? = null,
    val channel: String? = null,
    val channelConfigId: Int? = null,
    val description: String? = null,
    val status: String? = null,
    val creationDate: String? = null,
    val arn: String? = null,
    val endpoint: String? = null,
    val storageClass: String? = null,
    val versioningEnabled: Boolean? = null,
    val encryptionEnabled: Boolean? = null,
    val encryptionType: String? = null,
    val acl: String? = null
)

File: sdkwork-backend-sdk-csharp/Models/PlusOssBucketForm.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Backend.Models
{
    public class PlusOssBucketForm
    {
        public string? Name { get; set; }
        public string? Region { get; set; }
        public string? Channel { get; set; }
        public int? ChannelConfigId { get; set; }
        public string? Description { get; set; }
        public string? Status { get; set; }
        public string? CreationDate { get; set; }
        public string? Arn { get; set; }
        public string? Endpoint { get; set; }
        public string? StorageClass { get; set; }
        public bool? VersioningEnabled { get; set; }
        public bool? EncryptionEnabled { get; set; }
        public string? EncryptionType { get; set; }
        public string? Acl { get; set; }
    }
}
